src/business_logic/transaction/objects/internal_invoke_function.rs

```diff
@@ -29,7 +29,8 @@ use crate::{
 use felt::Felt;
 use num_traits::{ToPrimitive, Zero};
 
-pub(crate) struct InternalInvokeFunction {
+#[allow(dead_code)]
+pub struct InternalInvokeFunction {
```

Comment:
`#[allow(dead_code)]` shouldn't be needed in public code. Can this be removed without breaking the build?